src/auth/auth.ts

```diff
@@ -322,10 +322,11 @@ export class Auth extends Observable<SubscriberAuth> {
 
   /** store response from auth flow (accountName, accessToken, idToken) in localState */
   setAuthResult(authResponse: AuthResult) {
-    const { accessToken, idToken } = authResponse
+    const { accessToken, idToken, account } = authResponse
     if (!accessToken) throw Error('Cant setAuthResult. accessToken is missing')
     this._accessTokenHelper.setIdToken(null) // clear the existing idToken first (so set accessToken wont throw a mismatch when set)
     this.accessToken = accessToken // saves the token to localstorage
+    this._accessTokenHelper.setAccountName(account) // save accountName if account is passed in params
```

Comment:
Added this (`setAccountName` method & `_accountName` property) because sometimes the accessToken doesn't contains the `account` claim but the accountName is passed in params (AuthResult).